README.md

```diff
@@ -937,7 +937,7 @@ esetén szükségállapotot hirdethet ki.
 
 > *51. cikk*
```

Comment:
Ez a cikk az 53. cikk kellene, hogy legyen

Reply:
[net.jogtar.hu](https://net.jogtar.hu/jogszabaly?docid=a1100425.atv)

Reply:
valóban, valamit összekevertem

Reply:
A szövegkörnyezetbe jól illesztetted be. Valamikor korábban nem volt frissítve a cikk száma.